dev/webpack.config.js

```diff
@@ -13,7 +13,7 @@ const __DEV__ = process.env.NODE_ENV === 'development';
 const clean = arr => arr.filter(item => item !== false);
 
 module.exports = {
-  devtool: __DEV__ ? 'cheap-eval-source-map' : 'source-map',
+  devtool: 'source-map',
```

Comment:
Why did you introduce this change?

Reply:
From what I recall we had some issues with using this and breakpoints with @bobylito, I can roll this back if needed :)

Reply:
Hmm can we check together? Because this speeds-up re-build time for dev. I had no issues with breakpoints because webpack output sourcemaps also.